usaspending_api/recipient/v2/views/states.py

```diff
@@ -53,22 +53,30 @@ def obtain_state_totals(fips, year=None, award_type_codes=None, subawards=False)
         queryset = (
             matview_search_filter(filters, SummaryStateView)
             .values("pop_state_code")
-            .annotate(total=Sum("generated_pragmatic_obligation"), distinct_awards=StringAgg("distinct_awards", ","))
-            .values("distinct_awards", "pop_state_code", "total")
+            .annotate(
+                total=Sum("generated_pragmatic_obligation"),
+                distinct_awards=StringAgg("distinct_awards", ","),
+                face_value_of_loan_guarantee=Sum("face_value_loan_guarantee"),
+            )
+            .values("distinct_awards", "pop_state_code", "total", "face_value_loan_guarantee")
```

Comment:
The summed value is called `face_value_of_loan_guarantee`, but the value you're calling in `values` and for the results is still `face_value_loan_guarantee`.